### PR TITLE
build/configs/rtl8721csm: Modify BD RAM kregion size for trustzone

### PR DESCRIPTION
**Pull request by sangam-swami**
Modify size of kregion to accommodate trustzone region as well.
Trustzone uses 16KB from the BD RAM.

Signed-off-by: Sangamanatha <sangam.swami@samsung.com>

### CODE REVIEW

**Comment by sunghan-chang** [comment]
Where is Trustzone located in BD RAM? Is it 16KB from end of BD RAM?

**Comment by sangam-swami** [comment]
> Where is Trustzone located in BD RAM? Is it 16KB from end of BD RAM?
For trustzone  rlx8721d_img2_ns.ld is used, so we can modify defconfig to avoid overlapping

BD_RAM_NS (rwx)  :      ORIGIN = 0x10005000, LENGTH = 0x10077000 - 0x10005000   /* MA    IN RAM NS: 456K */
BD_RAM_S (rwx)  :       ORIGIN = 0x10077000, LENGTH = 0x1007B000 - 0x10077000   /* ROM: 16k */

File: build/configs/rtl8721csm/loadable_apps/defconfig [hunk 1]
```diff
@@ -197,7 +197,7 @@ CONFIG_BOOT_RUNFROMFLASH=y
 # Boot Memory Configuration
 #
 CONFIG_RAM_KREGIONx_START="0x10005000,0x02000000,"
-CONFIG_RAM_KREGIONx_SIZE="487424,3145728,"
+CONFIG_RAM_KREGIONx_SIZE="466944,3145728,"
```
Comment (sunghan-chang) [inline build/configs/rtl8721csm/loadable_apps/defconfig:200]: 487,424-466,944 = 20,480 = 20KB != 16KB

Comment (sangam-swami) [reply build/configs/rtl8721csm/loadable_apps/defconfig:200]: sorry it has 4KB reserved for non secure callable region as well. so total 20KB has to be excluded.

BD_RAM_NS (rwx)  :      ORIGIN = 0x10005000, LENGTH = 0x10077000 - 0x10005000 /* MAIN RAM NS: 456K */
BD_RAM_S (rwx)  :       ORIGIN = 0x10077000, LENGTH = 0x1007B000 - 0x10077000   /* MAIN RAM S: 16K */
BD_RAM_NSC (rwx)  :  ORIGIN = 0x1007B000, LENGTH = 0x1007C000 - 0x1007B000   /* MAIN RAM NSC: 4K */

Comment (sunghan-chang) [reply build/configs/rtl8721csm/loadable_apps/defconfig:200]: Could you add this content in commit description? (I don't mean PR description)
```
BD_RAM_NS (rwx) : ORIGIN = 0x10005000, LENGTH = 0x10077000 - 0x10005000 /* MAIN RAM NS: 456K /
BD_RAM_S (rwx) : ORIGIN = 0x10077000, LENGTH = 0x1007B000 - 0x10077000 / MAIN RAM S: 16K /
BD_RAM_NSC (rwx) : ORIGIN = 0x1007B000, LENGTH = 0x1007C000 - 0x1007B000 / MAIN RAM NSC: 4K */
```

Comment (sangam-swami) [reply build/configs/rtl8721csm/loadable_apps/defconfig:200]: added in commit description

Comment (sunghan-chang) [reply build/configs/rtl8721csm/loadable_apps/defconfig:200]: Sorry for late reply. Don't we need to modify this for all of defconfigs?

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**